manifest.json

```diff
@@ -14,6 +14,7 @@
     "open_in_tab": true
   },
   "permissions": [
```

Comment:
This should be an optional permission. This is an optional feature and I would absolutely uninstall the extension if it *required* the `contextMenus` permission.

Adding required permissions forces Chrome to disable the extension after an update as well, and using an optional permission would cut down on the "My AS stopped working. Fix it!" posts after the update.

Reply:
Done! Could you please test this to verify? I had some issues testing.